### PR TITLE
Loop and Learn Edits

### PR DESCRIPTION
**Pull request by tmhastings**
This PR has been replaced by PR #52

### CODE REVIEW

**Comment by marionbarker** [comment]
Still in draft form so will not approve. Adding for your consideration. 

With LoopDocs, we use automatic translations via google translate. To make the automatic translations more robust, there are some language choices we make (more formal languages, fewer colloquial choices). One I noticed when looking at these diffs is the use of `carbs` instead of `carbohydrates`. I'm not sure how translations will be handled for Trio.

File: docs/EN/Configuration/Devices.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 Adding devices to Trio is easy. You can only have one pump and one CGM active simultaneously. If you want to switch to another pump or CGM, delete the current device first. When you select a device, you will be guided through its setup with an on-screen guide.
 
 ## Pump
-This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](../Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
+This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
```
Comment (marionbarker) [inline docs/EN/Configuration/Devices.md]: This link was correct before. Perhaps your reviewers were looking at an outdated version of the docs.
You must go up one level from Configuration and then select Getting-Started, so `(../Getting-Started/pump.md)` is correct.

File: docs/EN/Configuration/Devices.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 
 During pump setup, you will be asked to choose your insulin type. This will determine your duration of insulin action and peak action time. <i>These settings can be changed later, <b>though it is not recommended.</i></b>
 
 ## CGM
-This section asks you to choose your glucose data source. Most options are self-explanatory. For more information on compatible CGMs, please go to [CGMs under Settings](../settings/devices/cgm.md).
+This section asks you to choose your glucose data source. Most options are self-explanatory. For more information on compatible CGMs, please visit [CGMs under Settings](settings/devices/cgm.md).
```
Comment (marionbarker) [inline docs/EN/Configuration/Devices.md]: Ditto. `(../settings/devices/cgm.md)` looks correct to me.

File: docs/EN/Configuration/transition-qa.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ meaning you can expect Trio to behave the same way AndroidAPS does generally.
   <summary><b>Should I switch from AndroidAPS to Trio?</b></summary>
 - If you have been successful with AndroidAPS but prefer the Apple ecosystem, you can easily transition to Trio. - If you've struggled to get stable BG with AndroidAPS, you may have the same issues with Trio.
   
-Take some time to read the docs and fine-tune your settings. To get help from other users, join [Discord](https://discord.gg/dbe5Twav8D) or [Facebook](https://www.facebook.com/groups/1351938092206709/). Once your settings are in order, consider switching to Trio.
+Take some time to read the docs and fine-tune your settings. To get help from other Trio users, join [Discord](https://discord.gg/dbe5Twav8D) or [Facebook](https://www.facebook.com/groups/1351938092206709/). Once your settings are in order, consider switching to Trio.
```
Comment (marionbarker) [inline docs/EN/Configuration/transition-qa.md]: Is this the link we want in the docs. (Asking out of ignorance.)

File: docs/EN/Configuration/transition-qa.md [hunk 4]
```diff
@@ -136,21 +136,22 @@ Here's the recommendation:
...
+Are you using a commercial closed-loop system like Insulet's Omnipod 5, Tandem's Control IQ, Medtronic's 780G, etc.? If you're happy, stay if you're not getting the desired results; read through the chapter [Coming from iOS Loop](#coming-from-ios-loop) chapter to see the main issues Trio solves.
 
 ## I Want to Try Trio. How Do I Proceed?
 
 - Read the documentation. 
 - Build the app. 
 - Go through all the settings and tap the setting name to get an explanation of what each setting does.
-If you have any questions, feel free to reach out on [Discord](https://discord.gg/fCY5svg4) or Facebook. But please read the docs first :)
+
+If you have any questions, feel free to reach out on [Discord](https://discord.gg/fCY5svg4) or [Facebook](https://www.facebook.com/groups/diytrio). The best practice is to read the docs first, then ask the community any questions you have.
```
Comment (marionbarker) [inline docs/EN/Configuration/transition-qa.md]: discord link?

File: docs/EN/settings/devices/cgm.md [hunk 5]
```diff
@@ -13,25 +13,25 @@ CGM selection includes the following:
 While using Nightscout as a CGM is an option, it should be avoided if possible because it will not keep Trio running in the background like other CGM options. You will have to open Trio manually to make it run loop cycles.
 
 ## xDrip (xDrip4iOS)
-To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](../../../operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
+To use xDrip4iOS as a CGM source, you must build it yourself with the same Apple Developer account you used to build your Trio app. You cannot use Shuggah or a version distributed by someone else's TestFlight. Please see the following for instructions on how to build xDrip4iOS yourself: [link](operate/build.html#xdrip4ios-or-glucose-direct-as-cgm-source)
```
Comment (marionbarker) [inline docs/EN/settings/devices/cgm.md:16]: I think this link should be (../operate/build.md#xdrip4ios-or-glucose-direct-as-cgm-source) (or at least that's the format that would be used in mkdocs.

File: docs/EN/settings/devices/pump.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
 # Pump
-This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](../../intro/requirements/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's bluetooth signals and the pump's radiocommunication.
+This section allows you to select your insulin delivery system. For more information on supported pumps, see the following: [link](Getting-Started/pump.md). Older Omnipod and Medtronic devices require a Rileylink to translate communication between your phone's Bluetooth signals and the pump's radiocommunication.
```
Comment (marionbarker) [inline docs/EN/settings/devices/pump.md:2]: I think this link should be {../../Getting-Started/pump.md)

File: docs/EN/Configuration/Devices.md (hunk 1 above)
Comment (tmhastings) [reply docs/EN/Configuration/Devices.md]: I thought so as well, but the link didn't work with the "../", but it did when I removed it. I will add it back and see if the error persists.

**Pull request closed without merging**